dawn/examples/tutorial/README.md

```diff
@@ -0,0 +1,106 @@
+# Getting Started using GTClang & dawn
+
+In this tutorial the basic usage of **GTClang** will be demonstrated using a simple example. To follow this tutorial, please make sure that you compiled **GTClang** with the `GTCLANG_ENABLE_GRIDTOOLS=ON` flag. The readme in the gtclang subdirectory has instructions on how to do that. We will compile and execute the same stencil three times: once starting from a stencil written with the **GTClang** DSL, once starting by using Python to write SIR, and once handing over SIR to dawn using C++.
+
+## Writing a Stencil in the GTClang SIR and Compiling the Stencil
+
+For the purpose of this exercise, we will write a simple Finite Difference Stencil to find the Laplacian of a function. In **GTClang**, this can be achieved using very few lines of code as demonstrated in `laplacian_stencil.cpp`:
+
+```
+globals {
+  double dx;  //grid spacing
+};
+
+stencil laplacian_stencil {
+  storage_ij out_field;
+  storage_ij in_field;
+  Do() {
+    vertical_region(k_start, k_end) {
+	    out_field[i,j] = (-4*in_field + in_field[i+1] + in_field[i-1] + in_field[j-1] + in_field[j+1])/(dx*dx);
```

Comment:
if we're using `[i,j]` in outfield we should probably be consistent and use it everywhere so also write `[i,j-1]` etc

Reply:
and if we do that, maybe add a sentence in the end saying there is a simplification

Reply:
Good idea. I added that.